shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -215,6 +226,9 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
 #ifndef __X64_SHELL__
   // A plugin that implements Tizen app_control channels.
   std::unique_ptr<AppControlChannel> app_control_channel_;
+
+  // A plugin that implements the Tizen window channel.
+  std::unique_ptr<WindowChannel> window_channel_;
```

Comment:
Please sort the channels in alphabetical order.

Reply:
Done.